microsoft-edge/devtools-guide-chromium/device-mode/index.md

```diff
@@ -176,6 +178,11 @@ If the mobile device option that you need isn't included on the default list, yo
 
    ![Adding a custom device](./index-images/device-mode-toggle-device-toolbar-settings-emulated-devices-add.png)
 
+1. Click **Add** to add the new custom device. The new device should now be available in the **Device** list.
+
+To remove a custom device:
+
+1. Under **Emulated Devices**, in the **Devices** list, hover over the custom device name and click the trash icon to delete the custom device.
```

Comment:
```suggestion
1. Under **Emulated Devices**, in the **Devices** list, hover over the custom device name, and then click the trash icon.
```
neither term is in style guide
https://learn.microsoft.com/en-us/search/?scope=Style%20guide&terms=trash
https://learn.microsoft.com/en-us/search/?scope=Style%20guide&terms=wastebasket

https://ahdictionary.com/word/search.html?q=trashcan